### PR TITLE
Added Msn Emojis (Windows Live Messenger) 

### PR DESCRIPTION
**Pull request by tahaipek**
Added Msn Emojis. (Windows Live Messenger) 
- Be right back 
- Cigarette 
- High 5 
- Xbox 
- Turtle 
- Crossed fingers 
- Age, sex, location 
- Handcuffs 
- Smile 
- Grin 
- Suprised 
- Tongue smile 
- Wink 
- Sad 
- Confused 
- Disappointed 
- Crying 
- Shy 
- Cool 
- Angry 
- Angel 
- Devil 
- Don't tell 
- Teeth 
- Dull 
- Sarcastic 
- Telling secrets 
- Sick 
- Don't know 
- Thinking 
- Party 
- Rolling eyes 
- Drowsy 
- Coffee 
- Thumb up 
- Thumb down 
- Beer 
- Cocktail 
- Girl 
- Guy 
- Guy hug 
- Girl hug 
- Bat 
- Cake 
- Heart 
- Broken heart 
- Kiss 
- Present 
- Rose 
- Wilted rose 
- Camera 
- Film 
- Cat 
- Dog 
- Phone 
- Lightbulb 
- Music note 
- Sleeping moon 
- Star 
- Envelope 
- Clock 
- MSN Messenger icon 
- Snail 
- Black sheep 
- Plate 
- Pizza 
- Soccer 
- Car 
- Airplane 
- Umbrella 
- Island with palm tree 
- Computer 
- Mobile phone 
- Storm 
- Lightning 
- Money

### CODE REVIEW

File: packs/msn.yaml [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+  - src: http://www.msgweb.nl/emoticons/58_58.gif
+    name: msnsoccer
+  - src: http://www.msgweb.nl/emoticons/59_59.gif
+    name: msncar
+  - src: http://www.msgweb.nl/emoticons/60_60.gif
+    name: msnairplane
+  - src: http://www.msgweb.nl/emoticons/61_61.gif
+    name: msnumbrella
+  - src: http://www.msgweb.nl/emoticons/62_62.gif
+    name: msnıslandwithpalmtree
```
Comment (hugovk) [inline packs/msn.yaml]: > Error: custom emoji names can only contain lower case letters, numbers, dashes and underscores: msnıslandwithpalmtree

https://travis-ci.org/hugovk/emojipacks/builds/198613178#L964

That `ı` should be an `i`.

Comment (hugovk) [inline packs/msn.yaml:146]: > Error: custom emoji names can only contain lower case letters, numbers, dashes and underscores: msnıslandwithpalmtree

https://travis-ci.org/hugovk/emojipacks/builds/198613178#L964

That `ı` should be an `i`.

**Review by hugovk — CHANGES_REQUESTED** [review]

**Comment by zachfeldman** [comment]
@tahaipek if you can make the changes @hugovk suggested, I will merge this!

**Comment by zachfeldman** [comment]
Hey there @tahaipek  , I'm closing this PR for now but feel free to open it back up later on if you'd like if it's still relevant!

**Pull request closed without merging**